src/contexts/AuthContext.js

```
@@ -42,7 +42,37 @@ export const AuthProvider = ({ children }) => {
+          // If userId is not in the response but id is, use id as userId
+          localStorage.setItem('userId', userData.id.toString());
+          console.log('Stored user ID from id field:', userData.id);
+        } else if (userData.userId) {
+          // Verify userId is stored correctly
+          console.log('User ID from response:', userData.userId);
+        } else {
+          // If no user ID in response, try to get it from getCurrentUser
+          console.warn('No user ID in sign-in response, attempting to fetch from getCurrentUser');
+          try {
+            const currentUserData = await AuthService.getCurrentUser();
+            if (currentUserData && currentUserData.id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)